usecases/ingestion_usecase.go

```diff
@@ -27,21 +27,23 @@ import (
 )
 
 const (
-	batchSize = 1000
+	csvIngestionBatchSize        = 1000
+	DefaultApiBatchIngestionSize = 100
 )
 
 type IngestionUseCase struct {
-	transactionFactory  executor_factory.TransactionFactory
-	executorFactory     executor_factory.ExecutorFactory
-	enforceSecurity     security.EnforceSecurityIngestion
-	ingestionRepository repositories.IngestionRepository
-	blobRepository      repositories.BlobRepository
-	dataModelRepository repositories.DataModelRepository
-	uploadLogRepository repositories.UploadLogRepository
-	GcsIngestionBucket  string
+	transactionFactory    executor_factory.TransactionFactory
+	executorFactory       executor_factory.ExecutorFactory
+	enforceSecurity       security.EnforceSecurityIngestion
+	ingestionRepository   repositories.IngestionRepository
+	blobRepository        repositories.BlobRepository
+	dataModelRepository   repositories.DataModelRepository
+	uploadLogRepository   repositories.UploadLogRepository
+	ingestionBucketUrl    string
+	batchIngestionMaxSize int
 }
 
-func (usecase *IngestionUseCase) IngestObjects(
+func (usecase *IngestionUseCase) IngestObject(
```

Comment:
just a comment to myself, but it's funny that git/github use a different "diff algorithm" than VSCode does